pom.xml

```diff
@@ -74,12 +74,12 @@
     <dependency>
       <groupId>com.github.genome-nexus.genome-nexus-java-api-client</groupId>
       <artifactId>genomeNexusPublicApiClient</artifactId>
-      <version>v1.1.0-beta</version>
+      <version>5a75010393fb0a1e42275cd6ddafdf696e1ad930</version>
```

Comment:
Is this really what we want for the version number? :thinking:

Reply:
@onursumer i'm using the commit hash https://github.com/genome-nexus/genome-nexus-java-api-client/commit/5a75010393fb0a1e42275cd6ddafdf696e1ad930 .
Maybe we could create a new version?

Reply:
@leexgh i created a new tag `v1.2.0-beta` that you can use

Reply:
@inodb thanks, already updated